### ISSUE TITLE
How to train a model?

### ISSUE THREAD

**Issue by wangzhiming68** [Leotju/MGAN#3]
I cannot find 'train.py',  should I use 'mmdetection/tools/train.py'?
Thanks!

**Comment by yustaub** [comment]
can you run the Testing step?

**Comment by wangzhiming68** [comment]
> can you run the Testing step?
No, because I cannot download pretrained model from PRC.

**Comment by yustaub** [comment]
> > can you run the Testing step?
> > Sorry, I cannot run the Testing step, because I cannot download pretrained model from PRC.
> > So I want to train the model myself.

I download the 50_65pth，but I still can't run the Testing step,I crete a folder named models,and put the 50_65.pth in it ,run the step ,and still error:launch.py: error: argument --nproc_per_node: invalid int value: 'models/50_65.pth'

**Comment by TWDH** [comment]
@Leotju Do you mind to share the train file in the github. I think this code is just for testing instead of training cause some of the file for training is missing. Very appriciate if you can release them XD

**Comment by RituYadav92** [comment]
@Leotju Same request.
Or, could you guys provide information on how loss is implemented or can provide some reference?

**Comment by hustXYY** [comment]
> I download the 50_65pth，but I still can't run the Testing step,I crete a folder named models,and put the 50_65.pth in it ,run the step ,and still error:launch.py: error: argument --nproc_per_node: invalid int value: 'models/50_65.pth'

Got same error...Have you solved it? Thanks a lot if informed

**Comment by ShaneGilroy** [comment]
Hi All,

Can you let me know which version of Pytorch, MMCV and MMDetection you used to get this running for basic test purposes? It is not compatible with the last two versions of mmdetection from what I gather.

**Comment by TonojiKiobya** [comment]
I am also waiting for the information where the loss function was implemented!
